source/whats-new.txt

```diff
@@ -74,6 +74,28 @@ What's New in 4.11
 
 .. New features of the 4.11 driver release include:
 
+- The following network address-related methods are deprecated and will be removed
+  in v5.0:
+
+  - The `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__
+    methods ``getSocketAddress()`` and ``getSocketAddresses()``.
+    
+    Instead of ``getSocketAddress()``, use the ``getByName()`` method.
+    ``getByName()`` is an instance method of the ``InetAddress`` class in the
+    ``java.net`` package.
+    
+    Instead of ``getSocketAddresses()``, use the ``getAllByName()`` method.
+    ``getAllByName()`` is an instance method of the ``InetAddress`` class in
+    the ``java.net`` package. 
+
+  - The `UnixServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/UnixServerAddress.html>`__
+    method ``getUnixSocketAddress()``.
+    
+    Instead of ``getUnixSocketAddress()``, use the constructor for the ``UnixSocketAddress``
+    class in the ``jnr.unixsocket`` package. Pass the file-system path name of the
+    UNIX socket file, as a ``String``, to the ``UnixSocketAddress`` constructor. By default,
+    MongoDB creates a UNIX socket file located at ``"/tmp/mongodb-27017.sock"``.
```

Comment:
Suggestion:

I think it could be better to omit the specific detail about passing a String and instead link to the API documentation. From those docs, it looks like the constructor can be either a String or File.

```suggestion
    Instead of ``getUnixSocketAddress()``, construct an instance of
    ``jnr.unixsocket.UnixSocketAddress``. Pass the full path of the UNIX
    socket file to the constructor. By default, MongoDB creates a UNIX
    socket file located at ``"/tmp/mongodb-27017.sock"``. To learn more
    about the ``UnixSocketAddress`` see the `UnixSocketAddress API documentation <https://www.javadoc.io/doc/com.github.jnr/jnr-unixsocket/latest/jnr/unixsocket/UnixSocketAddress.html>`__.
```

Reply:
Updated!